hack/test

```diff
@@ -61,6 +62,11 @@ if [ "$TEST_COVERAGE" = "1" ]; then
   coverageFlags="-coverprofile=/coverage/coverage.txt -covermode=atomic"
 fi
 
+runworker=""
+if [ -n "$TEST_WORKER" ]; then
```

Comment:
This should only be set if `TESTFLAGS` is not set as `go test` does not allow multiple `-run` . Or script should error if both are set.